spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    project
+    trait :finish_task do
```

Comment:
今回はtaskのstatusと同名のtrait名で伝わるかと思います。
```suggestion
    trait :done do
```